puppet/zulip/templates/postgresql/10/postgresql.conf.centos.template.erb

```diff
@@ -660,7 +660,7 @@ default_text_search_config = 'zulip.english_us_search'
 
 log_destination = 'stderr'
 logging_collector = on
-log_directory = '/var/log/postgresql'
+log_directory = 'log'
```

Comment:
Can you explain why this is a good idea?

Reply:
That's because with `/var/log/postgresql`, I encountered an error as described in https://chat.zulip.org/#narrow/stream/3-backend/topic/prod.20env.20on.20CentOS.207/near/812103. Using relative directory fixes the problem.